packages/components/src/HeaderBar/HeaderBar.component.js

```diff
@@ -153,12 +153,7 @@ function User({ name, firstName, lastName, getComponent, ...rest }) {
 		if (params.firstName && params.lastName) {
 			return (
 				<span className={classNames(theme['user-name'], 'user-name')}>
-					<span className={classNames(theme['user-firstname'], 'user-firstname')}>
-						{params.firstName}
-					</span>
-					<span className={classNames(theme['user-lastname'], 'user-lastname')}>
-						{params.lastName}
-					</span>
+					{params.firstName} {params.lastName}
```

Comment:
if `params.firsName` is empty then you will get an extra space

WDYT ? `{params.firstName ? params.firstName + ' ' : '' }{params.lastName}`

Reply:
Hmm I don't think so, there will be an extra space in the html, but not interpreted. Here it is interpreted because it is between 2 words.

Reply:
I just saw that my fix is not totally write. I'll remove the extra span. But in fact there is a test, if we have firstname *and* lastname it will display them. Else it will display name props